### PR TITLE
user.ltx 'fullscreen' to 'borderless' patch.

### PR DESCRIPTION
**Pull request by Red007Master**
There were multiple reports (my and my friend experienced it too) where `fullscreen` crashed game with fatal error, and thus I wrote this dirt patch.

### CODE REVIEW

**Comment by SeregaMP** [comment]
All Linux GAMMA Players are like this pull request!🫶

File: launcher/commands/install.py [hunk 1]
```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
 
     return arguments
 
+def replace_string_in_file(file_path, target_string, replacement_string):
```
Comment (Mord3rca) [inline launcher/commands/install.py:176]: Please use type hints in function definition:
`def replace_string_in_file(file_path: Path, target_string: str, replacement_string: str):`

File: launcher/commands/install.py [hunk 2]
```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
...
+    # Read the contents of the file
+    with open(file_path, 'r') as file:
```
Comment (Mord3rca) [inline launcher/commands/install.py:178]: file_path is a Path, so you can use `file_path.read_text()`

File: launcher/commands/install.py [hunk 3]
```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
...
+        file_contents = file.read()
+
+    # Replace the target string with the replacement string
+    modified_contents = file_contents.replace(target_string, replacement_string)
+
+    # Write the modified content back to the file
+    with open(file_path, 'w') as file:
```
Comment (Mord3rca) [inline launcher/commands/install.py:185]: file_path is a Path so you can use `file_path.write_text()`

In the end, this function is resumed at:
```python
file_path.write_text(
     file_path.read_text().replace(target_string, replacement_string)
)
```

**Review by Mord3rca — CHANGES_REQUESTED** [review]

File: launcher/commands/install.py (hunk 1 above)
Comment (Red007Master) [reply launcher/commands/install.py]: ready

File: launcher/commands/install.py (hunk 2 above)
Comment (Red007Master) [reply launcher/commands/install.py]: ready

File: launcher/commands/install.py (hunk 3 above)
Comment (Red007Master) [reply launcher/commands/install.py]: I will keep it on multiple lines to keep it more readable.

**Comment by Mord3rca** [comment]
Seems to be working, can you fix `flake8` defects and I'll merge:
```
./launcher/commands/install.py:176:1: E302 expected 2 blank lines, found 1
./launcher/commands/install.py:186:1: E302 expected 2 blank lines, found 1
./launcher/commands/install.py:260:5: E303 too many blank lines (2)
4     E302 expected 2 blank lines, found 1
2     E303 too many blank lines (2)
```

**Review by Mord3rca — APPROVED** [review]

**Pull request merged**